gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/LazyFormatterStep.java

```diff
@@ -56,4 +56,13 @@ public void readObject(ObjectInputStream input) throws IOException, ClassNotFoun
         FormatterStep serialized = (FormatterStep) input.readObject();
         delegate = () -> serialized;
     }
+
+    @Override
+    public void close() throws Exception {
```

Comment:
Is this method required by spotless now? If so, will upgrading PJF first before baseline bring in newer spotless and then error at runtime because this method is not implemented yet? Worth testing at least.